### ISSUE TITLE
[Accessibility: ACC-264.1] Button: UI name not read

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12587]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Name of the UI should be read out to the user.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Platform table](https://sap.github.io/fundamental-ngx/#/platform/table/basic)
Name of the button as ‘Back’ within the dialog ‘Filter by: Added ON’ is not read out on accessing the UI.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/90a2b406-b19d-484d-bcd0-987ea69e6b07">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 this should be addressed to the application. If you can create an isolated stackblitz example with the reported issue we can support you how to fix it.
